lib/msal-browser/src/app/ClientApplication.ts

```diff
@@ -155,6 +157,7 @@ export abstract class ClientApplication {
 
         // Initialize the token cache
         this.tokenCache = new TokenCache(this.config, this.browserStorage, this.logger, this.browserCrypto);
+        this.trackPageVisibilityWithMeasurement = this.trackPageVisibilityWithMeasurement.bind(this);
```

Comment:
What's the purpose of this?

Reply:
The purpose is to be able to use `trackPageVisibilityWithMeasurement` function as an event listener. I have added a comment.